core-plugins/src/test/java/co/cask/hydrator/plugin/batch/action/OracleExportActionTest.java

```diff
@@ -0,0 +1,170 @@
+      Assert.assertEquals("SSH authentication error when connecting to test@" + sshServer.getHost() + " on port " +
+                            sshServer.getPort(), e.getMessage());
+    }
+  }
+
+  @Test
+  public void testCheckWrongArgumentForQuery() throws Exception {
+    OracleExportAction.OracleExportActionConfig oracleExportActionConfig =
+      new OracleExportAction.OracleExportActionConfig(sshServer.getHost(),
+                                                      sshServer.getPort(), USER, PASSPHRASE, "system",
+                                                      "cask", "home", "sid", "/tmp/junit.csv",
+                                                      "select * from test", "csv");
+    OracleExportAction oracleExportAction = new OracleExportAction(oracleExportActionConfig);
+    MockActionContext mockActionContext = new MockActionContext();
+    try {
+      oracleExportAction.run(mockActionContext);
+    } catch (IllegalArgumentException e) {
+      Assert.assertEquals("Query should have ; at the end", e.getMessage());
+    }
+  }
+
+  @Test
+  public void testCheckWrongArgumentForFormat() throws Exception {
+    OracleExportAction.OracleExportActionConfig oracleExportActionConfig =
+      new OracleExportAction.OracleExportActionConfig(sshServer.getHost(),
+                                                      sshServer.getPort(), USER, PASSPHRASE, "system",
+                                                      "cask", "home", "sid", "/tmp/junit.csv",
+                                                      "select * from test;", "format");
+    OracleExportAction oracleExportAction = new OracleExportAction(oracleExportActionConfig);
+    MockActionContext mockActionContext = new MockActionContext();
+    try {
+      oracleExportAction.run(mockActionContext);
+    } catch (IllegalArgumentException e) {
+      Assert.assertEquals("Invalid format 'format'. Must be one of [csv, tsv, psv]", e.getMessage());
+    }
+  }
+
+  @Test
+  public void testOracleExportAction() throws Exception {
+    Path outputDir = dfsCluster.getFileSystem().getHomeDirectory();
+    OracleExportAction.OracleExportActionConfig oracleExportActionConfig =
+      new OracleExportAction.OracleExportActionConfig(sshServer.getHost(),
+                                                      sshServer.getPort(), USER, PASSPHRASE, "system",
+                                                      "cask", "home", "sid",
+                                                      outputDir.toUri().toString() + "/tmp/OracleExportJunitTemp.csv",
+                                                      "select * from test;", "csv");
+    OracleExportAction oracleExportAction = new OracleExportAction(oracleExportActionConfig);
+    MockActionContext mockActionContext = new MockActionContext();
+    oracleExportAction.run(mockActionContext);
+    Assert.assertTrue(fileSystem.exists(new Path(outputDir.toUri().toString() + "/tmp/OracleExportJunitTemp.csv")));
```

Comment:
so we're not checking the contents of the file, or even reading any data from oracle, just making sure that the exported file exists?